__tests__/endpoints.test.js

```diff
@@ -96,3 +96,43 @@ describe("/api", () => {
       });
   });
 });
+
+describe('/api/articles/:article_id', () => {
+    test('test for 200 status on completion', () => {
+        return request(app).get(`/api/articles/3`).expect(200);
+    });
+    test('should have the correct article by id', () => {
+        return request(app).get(`/api/articles/1`).expect(200).then((result) => {
+            expect(result.body).toEqual([{
+                article_id : 1,
+                title: 'Living in the shadow of a great man',
+                topic: 'mitch',
+                author: 'butter_bridge',
+                body: 'I find this existence challenging',
+                created_at: '2020-07-09T20:11:00.000Z',
+                votes: 100,
+                article_img_url:
+                  'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+              }])
+        });
+    });
+    test('should have all 8 properties', () => {
```

Comment:
This test is a better version of the one on line 104 - Object containing allows for later expansion and is a more flexible method than .toEqual 

If you only want to assert the specific id on 104 that would be a good test, but you should use .toHaveProperty not .toEqual ({entire Object})